api/api/models/logs.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+from api.api.models.base_models import BaseModel
+
+
+class APIUseCase(models.TextChoices):
+    """
+    Choices for the usecase field in the APILog model
+    """
+
+    NORMAL = "Normal", "Normal"
+    INTERNAL_DEMO = "Internal Demo", "Internal Demo"
+
+
+class APILog(BaseModel):
+    endpoint = models.TextField()
+    http_method = models.CharField(max_length=50)
```

Comment:
This is actually a discrete set of options, no?
Also idk if we gain that much value from this field tbh, the kind of API request we want to log is almost all POST

Reply:
We can have it default to POST, but I think it might be something that we might want in the future, but ofc we can't be sure of that. But defaulting it isn't really that helpful I guess since all of these records will be generated in one spot